include/sorting_network.h

```diff
@@ -0,0 +1,88 @@
+/*********************                                                        */
+/*! \file sorting_network.h
+** \verbatim
+** Top contributors (to current version):
+**   Makai Mann
+** This file is part of the smt-switch project.
+** Copyright (c) 2020 by the authors listed in the file AUTHORS
+** in the top-level source directory) and their institutional affiliations.
+** All rights reserved.  See the file LICENSE in the top-level source
+** directory for licensing information.\endverbatim
+**
+** \brief Implements a symbolic sorting network for boolean-sorted variables.
+**
+**/
+
+#pragma once
+
+#include "smt.h"
+
+namespace smt {
+
+/** \class SortingNetwork
+ *         Implements a symbolic sorting network for boolean-sorted variables.
+ *         All methods are const so it can be re-used for terms from the same
+ *         solver.
+ *         Does not return the actual terms, but is used to determine how many
+ *         variables are assigned to true symbolically.
```

Comment:
But it does return a list of terms... Can you clarify the comment (in the comment)?

Reply:
Yeah that was unclear, clarified in 03a6607a7a225e4b96a42cddd25fd335aed8df3d.